src/update/UpdateManager.h

```diff
@@ -0,0 +1,57 @@
+/*!********************************************************************
+ Audacity: A Digital Audio Editor
+
+ @file UpdateManager.h
+ @brief Declare a class that managing of updates.
+
+ Anton Gerasimov
+ **********************************************************************/
+#pragma once
+
+#include "VersionId.h"
+#include "VersionPatch.h"
+#include "UpdateDataParser.h"
+
+#include "Project.h"
+#include "Prefs.h"
+
+#include <wx/string.h>
+#include <wx/event.h>
+#include <wx/timer.h>
+
+/// A class that managing of updates.
+/**
+    Opt-in request and show update dialog by the scheduled time.
+    Have a built-in check that allows avoiding multiplying update notifications
+    when multiple Audacity windows are shown.
+*/
+class UpdateManager final : public wxEvtHandler
+{
+public:
+    UpdateManager(AudacityProject& project);
+    ~UpdateManager();
+
+    void getUpdates();
```

Comment:
The convention for naming member functions in Audacity is CamelCase, but also capitalizing the first letter.

Reply:
@Paul-Licameli just I little bit confused, could you clarifying this moment please. Okay, we have two style, lets nameing it as:
 * UpperCamelCase - `SomeMethodName()`
 * lowerCamelCase - `someMethodName()`

How we should naming:
 1 - local methods? (UpperCC or lowerCC)
 2 - static methods? (UpperCC or lowerCC)

Reply:
We prefer UpperCamelCase always.

Sometimes there are exceptions -- such as when you make a container class and must name members begin() and end() so that they work right of : in the range-for statement.